mos-platform/c128/init-mmu.S

```diff
@@ -0,0 +1,15 @@
+.section .bss,"aw",@nobits
+__mmusave:
+  .fill 1
+
+; MMU initialization needs to complete before any CRT initialization
+.section .init.010,"axR",@progbits
+        lda $FF00
```

Comment:
Instead, we can include `c128.inc` and do this symbolically; that should make the logic here very obvious

Reply:
That required actually fixing all the include files, but is now fixed.